packages/react-storage/src/components/StorageBrowser/controls/types.ts

```diff
@@ -31,10 +31,19 @@ interface TableData {
   isLoading?: boolean;
 }
 
+interface PaginationData {
+  currentPage: number;
+  disableNext: boolean;
+  disablePrevious: boolean;
+  handlePaginateNext: () => void;
+  handlePaginatePrevious: () => void;
+}
+
 export interface ControlsContext {
   data: {
     taskCounts?: TaskCounts;
     tableData?: TableData;
+    pagination?: PaginationData;
```

Comment:
Pagination and task counts never appear in the same view?

Reply:
Can you elaborate on this? I'm not sure if I follow